tests/copp/conftest.py

```diff
@@ -27,3 +30,15 @@ def pytest_addoption(parser):
         default="cold",
         help="reboot type such as cold, fast, warm, soft"
     )
+
+
+@pytest.fixture(autouse=True, scope="module")
```

Comment:
Changes look good. Please share the results from a non backend testbed as well

Reply:
Passes on frontend T0 
`================================================ 10 passed, 1 warning in 1491.24s (0:24:51) ================================================`